### DISCUSSION TITLE
凤凰网某些地方字体不变更，懂车帝二手车页面乱码

### DISCUSSION THREAD

**Discussion by jiaminxun** [F9y4ng/GreasyFork-Scripts#78] [问答专区(Question / Answer)] [answered]
凤凰网某些地方字体不变更，懂车帝二手车页面乱码，发图
![凤凰网](https://user-images.githubusercontent.com/57553009/160273454-03e662e3-fdcc-4792-852b-83de9ca93738.jpg)
![懂车帝](https://user-images.githubusercontent.com/57553009/160273460-0a0ee907-9da7-4564-8c0a-277c6676603f.jpg)

**Comment by F9y4ng** [comment, marked as answer]
**凤凰网**首页的标题，顶部的几条标题是在CSS样式里直接使用!important高优先级，所以新字体应用无效。

***

**懂车帝**，介绍或价格乱码问题：

在**需要渲染的网页元素**中，追加`:not([class*='tw-'])`，保存为站点独享即可。

**Reply by F9y4ng** [reply]
@jiaminxun 记得提问后在获得可用回答后勾选“mark as answer”，以方便其他人查看。

**Comment by Xia0xia0Xia0** [comment]
这种门户网站滥用important优先级实在是无语呀，百度也是各种随意的important优先定义，实际上完全没必要的滥用。

![2022-03-27](https://user-images.githubusercontent.com/101876873/160280978-6f674209-f6ef-4897-8357-532774383ba0.png)
